bin/cluster

```diff
@@ -172,7 +174,7 @@ if __name__ == '__main__':
 
     cluster = Cluster()
 
-    providers = ['gce', 'aws', 'libvirt']
+    providers = ['gce', 'aws', 'libvirt', 'os']
```

Comment:
Hmm, I'm afraid that 'os' is just too generic. That could be confused for meaning bare metal (what we today call 'byo'), or OpenShift (which wouldn't make sense, of course).

Could we just call this 'openstack' ?

Or is there another acronym that openstack uses, maybe ostack or something?